site-src/guides/tls.md

```diff
@@ -25,30 +25,33 @@ upstream connections is managed independently.
 
 Depending on the Listener Protocol, different TLS modes and Route types are supported.
 
-Listener Protocol | TLS Mode | Route Type Supported
---- | --- | ---
-TLS | Passthrough | TLSRoute
-TLS | Terminate | TCPRoute
-HTTPS | Terminate | HTTPRoute
+| Listener Protocol | TLS Mode    | Connection/s Affected   | Route Type Supported |
+|-------------------|-------------|-------------------------|----------------------|
+| TLS               | Passthrough | Downstream and Upstream | TLSRoute             |
+| TLS               | Terminate   | Downstream              | TCPRoute             |
+| HTTPS             | Terminate   | Downstream              | HTTPRoute            |
+| HTTPS             | Backend     | Upstream                | HTTPRoute            |
```

Comment:
I think the concepts of "downstream" and "upstream" are going to be very confusing here. I'd recommend copying the diagram you made for the GEP and adding it here along with the corresponding terminology. https://gateway-api.sigs.k8s.io/geps/gep-1897/#overview-what-do-we-want-to-do

Reply:
@robscott the concepts of downstream and upstream pre-exist in this document and the diagram.  I added a section called "Upstream", but there was already a section called "Downstream".    I can change "upstream" to "backend" here, but then what do I call what was formerly called "upstream" and not referencing the changes this introduces?

Reply:
Related comment, if I change "upstream" to "backend", shouldn't i also change "downstream" to "frontend'? Mixing the metaphors "downstream" and "backend" doesn't seem less confusing.

Reply:
🤦 you're right, I was not looking at the full deploy preview and related context, in which case the existing terminology you've used makes sense. 

I do really like the diagram you created for the GEP though, so if you wanted to make a version that uses the terminology in this doc (upstream and downstream), that could be a great upgrade on the current diagram here. (That's a non-blocking/optional suggestion though).

On further inspection, I would actually revert the change to this table. While "Passthrough" and "Terminate" are options on Listener TLS, "Backend" is not and is somewhat orthogonal.